schema/types/base/height.go

```diff
@@ -7,19 +7,35 @@ package base
 
 import "github.com/persistenceOne/persistenceSDK/schema/types"
 
-type height struct {
-	Value int64 `json:"height"`
-}
+//type height struct {
+//	Value int64 `json:"height"`
+//}
+
+var _ types.Height = &Height{}
 
-var _ types.Height = (*height)(nil)
+func (height *Height) Get() int64 {
+	return height.Value
+}
 
-func (height height) Get() int64 { return height.Value }
-func (height height) IsGreaterThan(compareHeight types.Height) bool {
+func (height *Height) IsGreaterThan(compareHeight types.Height) bool {
 	return height.Get() > compareHeight.Get()
 }
-func (height height) Equals(compareHeight types.Height) bool {
+
+func (height *Height) Equals(compareHeight types.Height) bool {
 	return height.Get() == compareHeight.Get()
 }
-func NewHeight(value int64) types.Height {
-	return height{Value: value}
+
+func NewHeight(value int64) Height {
+	return Height{Value: value}
```

Comment:
should this be height{}

Reply:
Pb.go files generate names beginning with Capital letter. So, we can change the name as soon as we generate a file.